pydantic_ai_slim/pydantic_ai/models/groq.py

```diff
@@ -47,10 +47,7 @@
 
 GroqModelName = Literal[
```

Comment:
The model names removed below were all marked as "shut down" (not just deprecated) in the page linked in the PR body.